### ISSUE TITLE
Final additions/tweaks to Places DB reorg

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#708]
Okay @jimmyrocks, just a few more things to sort out here before we can wrap up this boundaries database reorg:
1. Rename the `places_parks` database to `places_boundaries_v2` once the lakes and other non-boundary data is moved out into its own table (@mtaylorlong to follow with instructions for that).
2. We'd like to add one more field `pt_fill` (BOOLEAN) to the `parks_poly` table, which will be listed following `pt_render`. So, we'll need to regenerate the `parks_poly` table to get this field into the creation script. Please use the current table to populate the new version of the table including this new field, given I have made some changes in it today.
   - Set `pt_fill = 'true'` as default
   - Set `pt_fill = 'false' WHERE unit_id = 403 OR unit_id = 478 OR unit_id = 87  OR unit_id = 399 OR unit_id = 247 OR unit_id = 235 OR unit_id = 271 OR unit_id = 149 OR unit_id = 431 OR unit_id = 252`. This will turn off the fill for the following parks that overlap the larger NAMA and GWMP boundaries: wamo, wwii, coga, vive, linc, kowa, mlkm, frde, thje, lyba.
3. The field `pt_render` in the `parks_point` table is currently set to type TEXT. We need this to be set to type BOOLEAN.

Taylor has a couple more things to add, which he will follow with here. @mtaylorlong can you clarify the other database name as well, from my first bullet point (the DB not housing boundaries data).

Thanks @jimmyrocks!

**Comment by mtaylorlong** [comment]
Here are my additions:

I made some changes to the mapbox source in Jim's account that will need to be used for the next rendering:
- I updated the layer names to reflect the new naming
  - nps_places_boundaries_point
  - nps_places_boundaries_line
  - nps_places_boundaries_polygon
- I removed the great lakes labels completely from this source. I can add them to a new source I'm making that we can eventually store in a database called `places_custom` or something like that.
- @jimmyrocks will need to add a new line to the sql for the polygon layer to take advantage of our new field:
  - `"parks_poly"."pt_fill" AS "show_fill"`

That should do it! Thanks @jimmyrocks!
